extensions-core/kubernetes-extensions/src/main/java/org/apache/druid/k8s/discovery/K8sDruidNodeDiscoveryProvider.java

```diff
@@ -285,7 +282,7 @@ private void keepWatching(String namespace, String labelSelector, String resourc
               } else {
                 // Try again by starting the watch from the beginning. This can happen if the
                 // watch goes bad.
-                LOGGER.debug("Received NULL item while watching node type [%s]. Restarting watch.", this.nodeRole);
+                LOGGER.debug("Received NULL item while watching role[%s]. Restarting watch.", this.nodeRole);
```

Comment:
nit, since its debug log anyway
```suggestion
                LOGGER.debug("Received null item while watching role[%s]. Restarting watch.", this.nodeRole);
```